lib/components/PathProgress/PathProgress.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { cx } from '../../cva';
+
+export interface PathProgressProps {
```

Comment:
Maybe there should be `onClick` prop as these seems to be buttons. And it is propably wanted that user can click these path steps to jump to clicked one 🤔

Reply:
Yeah, I'm implementing right now

Reply:
Now it is not as a prop as I originally meant, but the current assumption is good and the best guess we have how this component is actually used.
Let's have it like this.